admin/onpage/class-onpage.php

```diff
@@ -0,0 +1,170 @@
+<?php
+/**
+ * @package WPSEO\Admin
+ */
+
+/**
+ * Handle the request for getting the onpage status
+ */
+class WPSEO_OnPage {
+
+	/**
+	 * The name of the user meta key for storing the dismissed status.
+	 */
+	const USERMETAVALUE = 'wpseo_dismiss_onpage';
+
+	/**
+	 * @var WPSEO_OnPage_Option The OnPage.org option class.
+	 */
+	private $onpage_option;
+
+	/**
+	 * Constructing the object
+	 */
+	public function __construct() {
+		// Only when AJAX isn't loaded.
```

Comment:
This line explains what is happening, not why. I would replace this with:
```
// We never want to fetch on AJAX request because doing a request is really slow.
```